stopwords.go

```diff
@@ -62,8 +64,10 @@ func (stop *StopWords) stopWordsCount(lang string, text string) wordStats {
 
 // SimpleLanguageDetector returns the language code for the text, based on its stop words; defaults to "en"
 func (stop StopWords) SimpleLanguageDetector(text string) string {
-	max := 0
-	currentLang := "en"
+	// max is set to 10 instead of 0; when the text is sparse, language detection maybe unstable;
+	// it may be better to fall back to the default language
+	max := 10
```

Comment:
Not sure what to set this `max` value to. When there is a lot of text, the differences in wordcounts by stopword list is reasonable, but it becomes problematic like in this edge case test `sparse body text` in `TestGetMetaLanguage` in `extractor_test`. When looking at the wordcounts across different stopword sets for that test, there were a fair number of languages with a count of 1 or 2 (and "de" with the highest count at 3 which is what ended up being returned by `StopWords.SimpleLanguageDetector()`).

Seems like we can do something better than what is here (i.e. setting the language in the language dectection to the language with the max count of stopwords)...

Reply:
Just trying to follow the logic here: `max` is essentially enforcing a minimum word length of a candidate for detection to take place?

If that's the case I'm okay with it, but might set it to a package variable?

Reply:
another option is a chi-squared test.. we could do a chi-sq test and look at the p-value to determine if there is 1 prominent language based on the stopwords. 
ex: 
```R
> chisq.test(c(en = 100, sp = 20, fr = 5))

	Chi-squared test for given probabilities

data:  c(en = 100, sp = 20, fr = 5)
X-squared = 125.2, df = 2, p-value < 2.2e-16
```

Reply:
update from IRL conversation: the chi-squared test might lead to high type 2 errors in languages with overlapping stopwords &mdash; Spanish/Italian, Russian/Ukranian, etc.